server/routes/adminDonors.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express';
+import { getDonorsByCampaignId } from '../controllers';
+import { authUser, authAdmin } from '../middlewares';
+
+const adminDonors = Router();
+
+adminDonors.get('/campaign/donors/:id', authUser, authAdmin, getDonorsByCampaignId);
```

Comment:
Here, i think if endpoint be like `/campaign/:id/donors` that would be more clear to read and understand